experiments/kompanion/cmd/export/export.go

```diff
@@ -116,20 +221,22 @@ func run(_ *cobra.Command, _ []string) {
 		log.Fatalf("Failed to get preferred resources: %s", err)
 	}
 
-	resources := []*metav1.APIResource{}
 	for _, apiResourceList := range apiResourceLists {
 		if !strings.Contains(apiResourceList.GroupVersion, "cnrm.") {
-			log.Printf("ApiResource %s group doesn't contain \"cnrm\"; skipping", apiResourceList.GroupVersion)
+			// todo acpana log debug level
```

Comment:
Just use `log := klog.FromContext(ctx)` and the `log.V(1).Info` (or whatever)

Reply:
Let me address the log verbosity as a follow on if that's ok! Right now the "debug" are off.